src/main/java/hu/dungeonhunter/characters/monsters/MonstersInterface.java

```diff
@@ -6,13 +6,25 @@
 public interface MonstersInterface {
     void getMonster();
     void getMonster(int hp);
+    void getMonster(int hp, int initiative, int accuracy, int defense, int numOfDices, int maxDamage);
     int getHp();
     void setHp(int hp);
     CharacterTypes getType();
     int getMonsterDamage();
     boolean isDefeat();
     int getInitiative();
-    void setInitiative(int Initiative);
-    int getFinalMonsterInitiation();
-    void monsterAttackInitiationCalculation();
+    void setInitiative(int initiative);
+    void setFinalInitiation(int finalInitiation);
+    int getFinalInitiation();
+    void attackInitiationCalculation();
+    int getAccuracy();
+    void setAccuracy(int accuracy);
+    int getDefense();
+    void setDefense(int defense);
+    int getFinalAccuracy();
+    void accuracyCalculation();
+    int getNumOfDices();
+    void setNumOfDices(int numOfDices);
+    int getMaxDamage();
+    void setMaxDamage(int maxDamage);
```

Comment:
Please delete unused interface methods (all that are grey)

Reply:
Once you have deleted finalAccuracy and finalInitiation variables, you can remove their setters and getters from the interface as well.

Reply:
I did it.